Libraries/libcar/Headers/car/Rendition.h

```diff
@@ -72,20 +72,49 @@ class Rendition {
         { return _format; }
     };
 
+public:
+    enum resize_mode {
```

Comment:
I actually find it strange that a person needs to use car_format.h to use libcar. I think over time libcar should replace all those enums and types with it's own, and keep car_format.h as internal header that describes just the file format, and not the C++ API to read and write the data.